cluster/v1beta2/0000_00_clusters.open-cluster-management.io_managedclustersets.crd.yaml

```diff
@@ -25,18 +25,18 @@ spec:
     name: v1beta2
     schema:
       openAPIV3Schema:
-        description: "ManagedClusterSet defines a group of ManagedClusters that user's
-          workload can run on. A workload can be defined to deployed on a ManagedClusterSet,
-          which mean: 1. The workload can run on any ManagedCluster in the ManagedClusterSet
+        description: "ManagedClusterSet defines a group of ManagedClusters that users'
```

Comment:
Suggestion: "ManagedClusterSet defines a group of ManagedClusters that you can run
          workloads on. You can define a workload to be deployed on a ManagedClusterSet. See the following options for the workload:
          - The workload can run on any ManagedCluster in the ManagedClusterSet
          - The workload cannot run on any ManagedCluster outside the ManagedClusterSet
          - The service exposed by the workload can be shared in any ManagedCluster
          in the ManagedClusterSet \n To assign a ManagedCluster to a certain
          ManagedClusterSet, add a label with the name `cluster.open-cluster-management.io/clusterset`
          on the ManagedCluster to refer to the ManagedClusterSet. You are not
          allowed to add or remove this label on a ManagedCluster unless you have an
          RBAC rule to CREATE on a virtual subresource of managedclustersets/join.
         To update this label, you must have the permission on both
          the old and new ManagedClusterSet."

Reply:
Updated.